api/src/main/java/io/opentelemetry/trace/DefaultTracer.java

```diff
@@ -56,30 +55,24 @@ public Scope withSpan(Span span) {
 
   @Override
   public Span.Builder spanBuilder(String spanName) {
-    return NoopSpanBuilder.create(this, spanName);
-  }
-
-  @Override
-  public HttpTextFormat<SpanContext> getHttpTextFormat() {
-    return HTTP_TEXT_FORMAT;
+    return NoopSpanBuilder.create(spanName);
```

Comment:
Changes in this class can be easily extracted.

Reply:
Well, we need for this specific change to rely on the `Propagators` being global, and to me this would look alien in the middle, so I'd rather not do it.